css/amp.css

```diff
@@ -831,3 +831,18 @@ amp-story .i-amphtml-loader {
 [amp-fx^="fly-in"] {
   visibility: hidden;
 }
+
+/**
+ * amp-addthis
+ */
+
+/**
+ * Make amp-addthis tool floating if user give data-widget-type
+ */
+
+amp-addthis[data-widget-type="floating"] {
+  position: fixed !important;
+  width:100% !important;
```

Comment:
@taojing10 do you need the `width` and `height` here? I feel like those should come from the `layout width height` as set on the component by page author.

Reply:
@aghassemi  from my test, if author put a fixed width like 320, and visitor use a bigger screen phone, out tool won't fit 100% of width. Thats why we want to force width to 100%.